cmd/in/main.go

```diff
@@ -55,17 +56,21 @@ func main() {
 
 	location := os.Args[1]
 
-	client := boshio.NewClient(progress.NewBar(), content.NewRanger(routines))
+	client := boshio.NewClient(progress.NewBar(), content.NewRanger(routines), inRequest.Source.ForceRegular)
 
 	stemcells, err := client.GetStemcells(inRequest.Source.Name)
 	if err != nil {
 		log.Fatalln(err)
 	}
 
-	stemcell, err := client.FilterStemcells(inRequest.Version.Version, stemcells)
-	if err != nil {
-		log.Fatalln(err)
+	filterFunc := func(stemcell boshio.Stemcell) bool {
+		return stemcell.Version == inRequest.Version.Version
+	}
```

Comment:
Again, I'd love to figure out how to push this logic down into the `FIlterStemcell` function itself instead of it living in main.